prepare_dir_html.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
```

Comment:
Probably you should add a comment somewhere here:
'# Prepare directory script for projects without database'